package.json

```diff
@@ -25,11 +25,11 @@
   },
   "homepage": "https://github.com/devongovett/browserify-istanbul",
   "dependencies": {
-    "istanbul": "~0.3.6",
     "minimatch": "^2.0.0",
     "through": "^2.3.4"
   },
   "devDependencies": {
+    "istanbul": "^0.4.2",
```

Comment:
Ok, this is fine then. Users will get an error at runtime if they don't install it.